schema/config-schema.json

```diff
@@ -100,6 +100,10 @@
                 "user": {
                     "id": "https://opencontainers.org/schema/bundle/process/user",
                     "type": "object",
+                    "required": [
+                        "uid",
```

Comment:
These are not required, because [they are POSIX-only][1] and `config-schema.json` also has to cover Windows.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc6/config.md#posix-platform-user